# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/2.6.10-sparc64.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The new copy_user stuff return value does yield warnings and thus we can not
compile the sparc dirs with -Werror ...

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.10/arch/sparc64/kernel/Makefile.vanilla	2005-02-01 20:35:16.000000000 +0100
+++ linux-2.6.10/arch/sparc64/kernel/Makefile	2005-02-01 20:35:30.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 EXTRA_AFLAGS := -ansi
-EXTRA_CFLAGS := -Werror
+EXTRA_CFLAGS := 
 
 extra-y		:= head.o init_task.o vmlinux.lds
 
--- linux-2.6.10/arch/sparc64/lib/Makefile.vanilla	2005-02-01 20:35:53.000000000 +0100
+++ linux-2.6.10/arch/sparc64/lib/Makefile	2005-02-01 20:35:58.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 EXTRA_AFLAGS := -ansi
-EXTRA_CFLAGS := -Werror
+EXTRA_CFLAGS := 
 
 lib-y := PeeCeeI.o copy_page.o clear_page.o strlen.o strncmp.o \
 	 memscan.o strncpy_from_user.o strlen_user.o memcmp.o checksum.o \
--- linux-2.6.10/arch/sparc64/mm/Makefile.vanilla	2005-02-01 20:35:39.000000000 +0100
+++ linux-2.6.10/arch/sparc64/mm/Makefile	2005-02-01 20:35:47.000000000 +0100
@@ -3,7 +3,7 @@
 #
 
 EXTRA_AFLAGS := -ansi
-EXTRA_CFLAGS := -Werror
+EXTRA_CFLAGS := 
 
 obj-y    := ultra.o tlb.o fault.o init.o generic.o extable.o
 
